### ISSUE TITLE
Replace PeakUtils?

### ISSUE THREAD

**Issue by TheSkyentist** [gbrammer/grizli#235]
Now that peakutils is incompatible with the newest SciPy versions and is no longer maintained, should it be replaced with an equivalent? Is `scipy.signal.find_peaks` a viable alternative?

**Comment by gbrammer** [comment]
Yes, that sounds like a good idea. The peakutils bits aren't particularly sophisticated, and are probably easily replaced with something like you mention.

**Comment by TheSkyentist** [comment]
It seems as though this Peakutils functionality was ported to scipy in
2017.  I’ll make sure the behavior is equivalent and make a PR with the
change.
https://github.com/scipy/scipy/issues/7393

**Comment by gbrammer** [comment]
Cool. If it's actually even the same algorithm that would be ideal. One useful bit I use from peakutils is the way it can find peaks with a user-specified minimum separation between them.

**Comment by TheSkyentist** [comment]
After some investigating and testing I can report that the algorithms are the same! PeakUtils does some internal rescaling of the data, but we can easily do that in-situ. The conversion between the two is straightforward:

```python
# PeakUtils Implementation
indices = peakutils.indexes(
        y,
        thres=t,
        min_dist=d
    )

# SciPy Signal Implementation
indices,_ = scipy.signal.find_peaks(
        y,
        height=t*(y.max()-y.min())+y.min(), # Rescale
        distance=d+1 # Note the off by one here!
    ) 
```
I'll submit a PR with the required changes.

**Comment by TheSkyentist** [comment]
Closed by #236
